modules/geo-layers/src/tile-layer/utils.js

```diff
@@ -141,9 +180,18 @@ export function getTileIndices({viewport, maxZoom, minZoom, zRange, extent, tile
   if (Number.isFinite(maxZoom) && z > maxZoom) {
     z = maxZoom;
   }
+  let transformedExtent = extent;
+  if (modelMatrix && extent && !viewport.isGeospatial) {
+    transformedExtent = transformBox(extent, modelMatrix);
```

Comment:
Should this be stored outside of this since it doesn't need to be run every time?